content/realtime/authentication.textile

```diff
@@ -112,6 +113,12 @@ Below is a rather contrived yet straightforward example that instances a Realtim
   AblyRealtime realtime = new AblyRealtime(options);
 ```
 
+h4(#token-upgrading). Upgrading the token on a connection
+
+In version 0.9 of the client libraries, the token that a connection uses can be upgraded on the fly. When a token is nearly expired, Ably will prompt the library to request a new token (using the @authUrl@ or @authCallback@). Once the library has a new token, it will upgrade the connection to use the new token. This all happens automatically and seamlessly.
```

Comment:
Technically the token could be issued by the client library itself if it has an API key, but I would agree that it's not relevant in most cases so you can leave like this.